libindy/src/domain/ledger/constants.rs

```diff
@@ -23,6 +23,7 @@ pub const GET_REVOC_REG_DELTA: &str = "117";
 pub const GET_VALIDATOR_INFO: &str = "119";
 pub const AUTH_RULE: &str = "120";
 pub const GET_AUTH_RULE: &str = "121";
+pub const AUTH_RULES: &str = "122";
```

Comment:
What's the difference between AUTH_RULE 120 and AUTH_RULES 122?

Reply:
AUTH_RULE send one specific rule
AUTH_RULES accepts and send vector of auth rules